core/node/base_token_adjuster/src/base_token_adjuster.rs

```diff
@@ -165,21 +163,27 @@ impl MainNodeBaseTokenAdjuster {
 
         tokio::time::sleep(sleep_duration).await;
     }
+
+    /// If no base token L1 address is set then the base token is ETH.
+    fn is_base_token_eth(&self) -> bool {
+        self.base_token_l1_address.is_none()
```

Comment:
```suggestion
        !self.base_token_l1_address.map_or(true, |v| v == L1_ETH_ADDRESS);
```

You also need to check if its the L1_ETH_ADDRESS

Reply:
100% -- made the token address non optional and comparing with the ETH shared bridge "address"